locales/en.json

```diff
@@ -55,13 +55,14 @@
     "address": "Address",
     "reasonForSharing": "Reason for sharing",
     "idType": "ID Type",
-    "id": "Id",
+    "id": "ID",
     "qrCodeHeader": "QR Code",
     "nationalCard": "National Card",
     "uin": "UIN",
     "vid": "VID",
     "enableVerification": "Activate",
     "profileAuthenticated": "Credentials are enabled for online authentication.",
+    "closedCardProfileAuthenticated": "Activated for online login",
```

Comment:
Why don't we change the `profileAuthenticated` value instead of adding new one ?

Reply:
The above scenario is the reason

Reply:
used the walletbinding label